### ISSUE TITLE
Change sentence in abstract

### ISSUE THREAD

**Issue by kbruegge** [gammapy/gammapy-v1.0-paper#108]
I did a double take while reading this sentence:

> Alongside the data, they will make available to the community software tools for their analysis.

Maybe rephrase to something along the lines of ?

> Alongside the data, their software tools for data analysis will be made available to a wider community.

**Comment by adonath** [comment]
Changed to: "Alongside the data, they will also make associated software tools available to a wider community."
